kong/db/dao/plugins/go.lua

```diff
@@ -444,18 +471,22 @@ end
 local get_plugin do
   local loaded_plugins = {}
 
+  local function get_plugin_info(name)
+    local cmd = string.format(
+        "bin/go-pluginserver -plugins-directory %q -dump-plugin-info %q",
+        kong.configuration.go_plugins_dir, name)
+
+    local d = assert(io.popen(cmd)):read("*a")
```

Comment:
don't wait for the GC to close the fd: store it in a variable and `close()` it explicitly.

Reply:
done